spec/support/issue.cr

```diff
@@ -0,0 +1,23 @@
+class Issue < BaseModel
+  COLUMN_SQL = "issues.id, issues.status, issues.role"
+
+  avram_enum Status do
+    Opened = 0
+    Closed = 1
+    Duplicated = 2
+  end
+
+  avram_enum Role do
+    Issue = 0
```

Comment:
What if we keep the values set here, but change them to be like 1, 2, 3 so we can showcase that you can override the values if you need?

Reply:
Sure, as they are just sent directly to enum this will work. I can do that later :)

Reply:
Fixed this now so resolve this :)